storage/storage-service/src/lib.rs

```diff
@@ -35,6 +35,102 @@ use storage_proto::proto::storage::{
 };
 use tokio::runtime::Runtime;
 
+use libra_secure_net::NetworkServer;
+use libra_types::{account_state_blob::AccountStateBlob, proof::SparseMerkleProof};
+use std::thread::{self, JoinHandle};
+use storage_proto::StartupInfo;
+
+/// Starts storage service with a given LibraDB
+pub fn start_storage_service_with_db_simple(
+    config: &NodeConfig,
+    libra_db: Arc<LibraDB>,
+) -> JoinHandle<()> {
+    let storage_service = StorageServiceSimple { db: libra_db };
+    start_storage_service_loop(config, storage_service)
+}
+
+/// Starts storage service according to config.
+pub fn start_storage_service_new_simple(config: &NodeConfig) -> JoinHandle<()> {
+    let storage_service = StorageServiceSimple::new(&config.storage.dir());
+    start_storage_service_loop(config, storage_service)
+}
+
+fn process_one_message(
+    network_server: &mut NetworkServer,
+    service: &StorageServiceSimple,
+) -> Result<(), Error> {
+    let request = network_server.read()?;
+    let response = service.handle_message(request)?;
+    network_server.write(&response)?;
+    Ok(())
+}
+
+fn start_storage_service_loop(
+    config: &NodeConfig,
+    service: StorageServiceSimple,
+) -> JoinHandle<()> {
+    let mut network_server = NetworkServer::new(config.storage.address);
```

Comment:
this is a conflict with the existing storage service. are you gonna migrate away from it entirely before using this ServerSimple in the node?

Reply:
Good point. My plan is to use both for now since your backup tool need GRPC support. Otherwise the old can be removed.